redos/7/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM centos:7 as prepare
+MAINTAINER Sergei Vorontsov "piligrim@rootnix.net"
+
+ENV REPO_URL=http://mirror.yandex.ru/redos/7.3/x86_64/os/ \
+    CHROOT_DIR="/chroot"
+
+RUN mkdir -p $CHROOT_DIR
+
+RUN for rpm in `curl -s $REPO_URL | grep 'redos-release' | sed 's/<.*>\(.*\)<\/.*>/\1/' | awk '{print $1}'`; do if [[ $rpm =~ redos-release-[0-9].[0-9]-([0-9]|[0-9][0-9]).el7.noarch.rpm ]]; then curl -o $CHROOT_DIR/redos-release.rpm $REPO_URL$rpm; fi; done
+
+#setup base system
+RUN rpm --root=$CHROOT_DIR --rebuilddb
+RUN rpm --root=$CHROOT_DIR --nodeps --quiet -i $CHROOT_DIR/redos-release.rpm
+RUN cp $CHROOT_DIR/etc/pki/rpm-gpg/RPM-GPG-KEY-RED-SOFT /etc/pki/rpm-gpg/RPM-GPG-KEY-RED-SOFT
+
+RUN yum -y --installroot=$CHROOT_DIR update
+
+RUN yum -y --installroot=$CHROOT_DIR install -y \
```

Comment:
How packages are chosen? Can we do `yum -y groupinstall 'Development Tools'` similarly to CentOS 7 image?

I think a short comment about motivation behind this package list would be helpful.

If the motivation is to install everything that is needed for building tarantool, running tests and create an RPM package, it lacks zlib-devel and python3-pyyaml at least.

Reply:
> <...> it lacks zlib-devel and python3-pyyaml at least.

If I run `make -f ~/packpack/pack/Makefile -C . BUILDDIR=~/build -j` (see https://github.com/tarantool/infra/issues/106) from inside the container and several more packages are installed: libunwind-devel (not a real dependency and I guess it'll be removed in https://github.com/tarantool/tarantool/issues/6998), perl-Test-Harness and python3-gevent.

It is okay per se. I just want to understand a rule how this list was created.

Reply:
Group packages Development Tools has 2 subgroups mandatory, default and optional. If I install all of them that docker images increases size to 1,7Gb from 0.8Gb. I checked witch packages needs for building tarantool and install them separately.

Reply:
I splitted general development packages from packages necessary to build gevent using pip. Added explanations as comments. Now it should be clear.